### PR TITLE
New Package: love-0.9.2

### PR DESCRIPTION
**Pull request by anarat**

### CODE REVIEW

File: srcpkgs/love/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+short_desc="A framework for writing 2D games in lua"
+maintainer="Caleb Pilon <CJP@cock.li>"
+license="zlib"
+homepage="https://love2d.org/"
+distfiles="https://bitbucket.org/rude/love/downloads/${pkgname}-${version}-linux-src.tar.gz"
+checksum="a995d062dc14e7944c239934b9343583153718774511d8625885aa57f2b5c172"
+makedepends="SDL2-devel devil-devel libopenal-devel LuaJIT-devel freetype-devel physfs-devel libmodplug-devel mpg123-devel libvorbis-devel"
+depends="SDL2 libopenal LuaJIT devil-devel freetype physfs libmodplug mpg123 libvorbis"
```
Comment (xtraeme) [inline srcpkgs/love/template]: - You don't need to specify the run-time dependencies, in ELF files they are detected by xbps-src automagically.
- There's no reason to provide a static library without any header, add `configure_args="--disable-static"`.
- There's no need to quote `pkgname` and `version`.

Comment (anarat) [inline srcpkgs/love/template]: Thank you for helping, i'm pretty new to github/linux :).

**Comment by xtraeme** [comment]
Please rebase to avoid merge and dup commits.

**Comment by xtraeme** [comment]
Please rebase and submit a single commit.

**Comment by Gottox** [comment]
merging all commits into one: 
```
git fetch https://github.com/voidlinux/void-packages.git;git rebase -i FETCH_HEAD
```

**Pull request closed without merging**